crates/router/src/core/payouts.rs

```diff
@@ -2549,6 +2581,44 @@ pub async fn make_payout_data(
         payouts::PayoutRequest::PayoutRetrieveRequest(_) => None,
     };
 
+    if payout_method_data.is_some() {
+        let async_move_profile_id = profile_id.clone();
+        let additional_pm_data = payout_method_data
+            .clone()
+            .async_and_then(|payout_method_data| async move {
+                helpers::get_additional_payout_data(
+                    &payout_method_data,
+                    &*state.store,
+                    &async_move_profile_id,
+                )
+                .await
+            })
+            .await;
+
+        let additional_payout_method_data = additional_pm_data
+            .as_ref()
+            .map(Encode::encode_to_value)
+            .transpose()
+            .change_context(errors::ApiErrorResponse::InternalServerError)
+            .attach_printable("Failed to encode additional payout method data")?;
+
+        let update_additional_payout_method_data =
+            storage::PayoutAttemptUpdate::AdditionalPayoutMethodDataUpdate {
+                additional_payout_method_data,
+            };
+
+        payout_attempt = db
```

Comment:
Is this added for update / confirm flows?

Reply:
Yes, this is added for update as well as confirm call